client-src/elements/form-field-specs.js

```diff
@@ -1391,6 +1409,16 @@ export const ALL_FIELDS = {
     </ul>`,
   },
 
+  'stage_description': {
+    type: 'textarea',
+    required: true,
+    label: 'Description',
+    help_text: html`
+      <p>
+        A brief description of the feature (1-2 sentences). Displayed publicly.
```

Comment:
We should clarify how this relates to the main feature description.  I.e., is this the one displayed on the origin trial console?

What if we didn't have this field and just used the main feature description on the OT console?

Reply:
> We should clarify how this relates to the main feature description. I.e., is this the one displayed on the origin trial console?

That's a good idea. I tried to make it more general description field in case we need to use it for other stages, but maybe it's best to make it more specific to origin trials. 🙂 

> What if we didn't have this field and just used the main feature description on the OT console?

After looking around at the active origin trials in production, it seems that this description is intended to be shorter and almost always varies from the description displayed in ChromeStatus, so I assumed we'd need a new field entirely